packages/react/src/components/Badge/BadgePlacement.tsx

```diff
@@ -0,0 +1,38 @@
+import cl from 'clsx/lite';
+import { type HTMLAttributes, forwardRef } from 'react';
+import type { DefaultProps } from '../../types';
+import type { MergeRight } from '../../utilities';
+export type BadgePlacementProps = MergeRight<
+  DefaultProps & HTMLAttributes<HTMLSpanElement>,
+  {
+    /**
+     * The placement of the badge
+     *
+     * @default top-right
+     */
+    placement?: 'top-right' | 'top-left' | 'bottom-right' | 'bottom-left';
+    /**
+     * Use when badge is floating to change the position of the badge
+     *
+     * @default rectangle
+     */
+    overlap?: 'circle' | 'rectangle';
+  }
+>;
+
+export const BadgePlacement = forwardRef<HTMLSpanElement, BadgePlacementProps>(
```

Comment:
What if we call this component `Badge.Position`? 

Was abit confusing on the eyes having "placement" double up with a prop also called the same. 

`<Badge.Placement placement="top-left" />`
vs
`<Badge.Position placement="top-left" />`

What do you guys think? @eirikbacker

Reply:
I don't have a lot of opinions on the component name here. 
I like `.Position`. We can also go for `.Floating`

Reply:
yeah, `.Floating` could also work 🤔

Reply:
Since this wrapper has to have a css class as well, I am leaning more towards `position`.. since this is what sets `position: relative`. 
It's just semantics, so most things are fine with me

Reply:
I renamed it to `.Position`